package.json

```
@@ -19,6 +19,7 @@
   "dependencies": {
     "@algolia/client-search": "^4.9.1",
     "@docusaurus/core": "^2.0.0-beta.20",
+    "@docusaurus/plugin-ideal-image": "^2.0.0-beta.21",
```

```suggestion
    "@docusaurus/plugin-ideal-image": "^2.0.0-beta.20",
```

I think it is better to use the same version as `@docusaurus/core` to assure no inconsistencies as they are developed together. And update the lock file by running `yarn` afterwards.